testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/ClientAuthSignedJWTTest.java

```diff
@@ -845,7 +845,9 @@ protected JsonWebToken createRequestToken(String clientId, String realmInfoUrl)
 
             int now = Time.currentTime();
             if (isClaimEnabled("issuedAt")) reqToken.issuedAt(now);
-            if (isClaimEnabled("expiration")) reqToken.expiration(now + getTokenTimeout());
+            // For the time being there's no getter for tokenTimeout in JWTClientCredentialsProvider
```

Comment:
Well, one is changing the implementation for the sake of a test, which is bad! The other is working around it in the test, which is perfectly fine. It's a test, if you want to test that scenario then you have plenty of opportunities to retrieve the expiration without modifying the implementation.